src/main/java/com/twcable/grabbit/cli/Utils.java

```diff
@@ -13,26 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.twcable.grabbit.cli
+package com.twcable.grabbit.cli;
 
-class ThreadedStreamProcessor extends Thread {
-    InputStream inputStream
-    def output = []
+class Utils {
 
-
-    ThreadedStreamProcessor(InputStream inputStream) {
-        this.inputStream = inputStream
+    /**
+     * Remove checked-ness from the exception. The same exception is still thrown (checked or unchecked), but this
+     * removes the compiler's checks.
+     */
+    @SuppressWarnings("RedundantTypeArguments")
+    public static <T extends Throwable> void throwSoft(T exp) {
+        throw Utils.<RuntimeException>uncheck(exp);
```

Comment:
If it does work, seems a bit clever. Do we really want to avoid compiler checks in this way? Seems at first glance like it could only serve to benefit us